### PR TITLE
Initial Cleanup

### PR DESCRIPTION
**Pull request by dayveed-repo**
changed some styles and did some modifications to some codes

### CODE REVIEW

File: config/firebase.js [hunk 1]
```diff
@@ -1,23 +0,0 @@
-import firebase from 'firebase/app';
```
Comment (Dami-Olus) [inline config/firebase.js:1]: Add this file back and remove from gitignore. I dont think there is any sensitive data

File: screens/Home.js [hunk 2]
```diff
@@ -66,13 +59,24 @@ const Home = () => {
...
+    useEffect(() => {
+      const fetchuserData = async () => {
+        await firestore.collection("users").doc(user.uid).get()
+              .then(doc => setusersFullname(doc.data().name))
+              .catch(error => console.log(error))
+      }
+      
+      fetchuserData()
+    }, [])
+
```
Comment (Dami-Olus) [inline screens/Home.js:71]: The screen keeps refreshing and I think it might be because of this code

Comment (Dami-Olus) [reply screens/Home.js:71]: Please check why the screen keeps refreshing

**Review by Dami-Olus — CHANGES_REQUESTED** [review]

File: screens/Home.js (hunk 2 above)
Comment (dayveed-repo) [reply screens/Home.js:71]: the useEffect hook has no dependency as such will only run after the page has loaded and it doesn't seem to refresh on my end. But I'll check the code to see what might potentially be causing the problem

File: config/firebase.js (hunk 1 above)
Comment (dayveed-repo) [reply config/firebase.js:1]: okay ill add that

**Review by Dami-Olus — APPROVED** [review]

**Pull request merged**